### PR TITLE
minor improvement of ClusterReceptionist loading

### PR DESCRIPTION
**Pull request by patriknw**
just some small boy scouting

### CODE REVIEW

File: akka-actor-typed-tests/src/test/scala/akka/actor/typed/ActorContextSpec.scala [hunk 1]
```diff
@@ -84,6 +84,8 @@ abstract class ActorContextSpec extends ActorTestKit with TypedAkkaSpecWithShutd
           case Renew(ref) ⇒
             ref ! Renewed
             behavior
+          case other ⇒
+            throw new RuntimeException(s"Unexpected message: $other")
```
Comment (patriknw) [inline akka-actor-typed-tests/src/test/scala/akka/actor/typed/ActorContextSpec.scala]: silence compiler warning

File: akka-actor-typed-tests/src/test/scala/akka/actor/typed/ActorContextSpec.scala [hunk 2]
```diff
@@ -531,7 +533,7 @@ abstract class ActorContextSpec extends ActorTestKit with TypedAkkaSpecWithShutd
       val probe = TestProbe[ActorRef[String]]()
       val actor = spawn(Behaviors.immutablePartial[String] {
         case (ctx, "message") ⇒
-          messages.ref ! (ctx.self, "received message")
+          messages.ref.tell((ctx.self, "received message"))
```
Comment (patriknw) [inline akka-actor-typed-tests/src/test/scala/akka/actor/typed/ActorContextSpec.scala]: silence compiler warning

File: akka-persistence-typed/src/test/scala/akka/persistence/typed/scaladsl/PersistentBehaviorSpec.scala [hunk 3]
```diff
@@ -27,7 +27,7 @@ object PersistentBehaviorSpec {
     }
 
     def saveAsync(metadata: SnapshotMetadata, snapshot: Any): Future[Unit] = {
-      state += (metadata.persistenceId -> (snapshot, metadata))
+      state = state.updated(metadata.persistenceId, (snapshot, metadata))
```
Comment (patriknw) [inline akka-persistence-typed/src/test/scala/akka/persistence/typed/scaladsl/PersistentBehaviorSpec.scala]: silence compiler warning

**Review by johanandren — APPROVED** [review]
LGTM, thanks!

**Review by chbatey — APPROVED** [review]
LGTM

**Pull request merged**